JavaScript/chapter01/1.4 - PalinPerm/rroque98_sol.js

```diff
@@ -0,0 +1,28 @@
+// Given a string, write a function to check if it is a permutation of a palindrome
+
+const isPalindromePermutation = (str) => {
+  const strNoSpaces = str.split(' ').join('');
+  const obj = {};
+  var oddCount = 0;
+  for (let char of strNoSpaces) {
+    if (obj[char] !== undefined) {
+      obj[char]++;
+    } else {
+      obj[char] = 1;
+    }
+  }
+  for (let key in obj) {
```

Comment:
since you are not using the key for anything, it might be better to do

```javascript
const frequencies = new Map();
// ...
for (let frequency of frequencies.values()) {
  if (frequency % 2 == 0) {
    continue;
  }
  oddCount++;
  if (oddCount > 1) {
    return false;
  }
}
```